spec/date.html

```diff
@@ -16,16 +16,22 @@ <h1>The Temporal.Date Constructor</h1>
       Subclass constructors that intend to inherit the specified Date behaviour must include a super call to the %Temporal.Date% constructor to create and initialize subclass instances with the necessary internal slots.</p>
 
     <emu-clause id="sec-temporal.date">
-      <h1>Temporal.Date ( _year_, _month_, _day_, _disambiguation_ )</h1>
+      <h1>Temporal.Date ( [ _year_ [ , _month_ [ , _day_ [ , _disambiguation_ ] ] ] ] )</h1>
       <p>
         When the `Temporal.Date` function is called, the following steps are taken:
       </p>
       <emu-alg>
         1. If NewTarget is *undefined*, then
           1. Throw a *TypeError* exception.
         1. Let _y_ be ? ToInteger(_year_).
```

Comment:
I guess we're now consistently relying on the property `ToInteger(undefined) === 0`. Please consider whether it's worth documenting that.

Reply:
You mean as a side note for clarity? I figured one could easily click through to the actual spec and it would be easy to miss the special "1" values if they looked visually the same as checking for "undefined" and assigning "0" values on the rest of the elements.